wp-content/themes/reconasia/template-parts/block-post.php

```diff
@@ -12,16 +12,26 @@
  */
 
 ?>
-
 <article <?php post_class('post-block post-block--post'); ?> id="post-<?php the_ID(); ?>">
+	<div class="post-block-right">
+		<?php
+			// if ( !is_front_page() ){
+				// echo get_the_post_thumbnail( get_the_ID(), 'large' );		
+			// }
+			echo get_the_post_thumbnail();
```

Comment:
Will you please make the thumbnail a link to the post and have the hover effect be applied to the title when you hover over the image?